package.json

```diff
@@ -62,9 +62,9 @@
   },
   "devDependencies": {
     "babel": "^5.8.23",
-    "babel-eslint": "^4.1.1",
+    "babel-eslint": "^6.1.2",
     "cucumber": "^0.9.2",
-    "eslint": "^1.10.3",
+    "eslint": "^3.4.0",
```

Comment:
Be aware that this requires node 4. And you need to adjust the rules.

http://eslint.org/docs/user-guide/migrating-to-3.0.0#dropping-support-for-nodejs--4